### PR TITLE
new tests to cover change_password script

### PR DESCRIPTION
**Pull request by geonexus**
### REVIEWERS
@chemaper @flopezag 

### DESCRIPTION
- new tests to cover change_password script

### CODE REVIEW

File: skuld/change_password.py [hunk 1]
```diff
@@ -27,15 +27,24 @@
...
-from utils.osclients import OpenStackClients
-
+from utils import osclients
+import os
 
 class PasswordChanger(object):
     """Class to change/reset the password of any user"""
-    def __init__(self):
+    def __init__(self, osclients_o=None):
         """constructor of the object. It expect the environment variables with
```
Comment (flopezag) [inline skuld/change_password.py]: It receives the ...

File: tests/units/test_change_password.py [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+class TestChangePassword(TestCase):
+    mock_session2 = MySessionMock()
+
+    def setUp(self):
+        self.OS_AUTH_URL = 'http://cloud.lab.fi-ware.org:4731/v2.0'
+        self.OS_USERNAME = 'user'
+        self.OS_PASSWORD = 'password'
+        self.OS_TENANT_NAME = 'user cloud'
+        self.OS_TENANT_ID = '00000000000000000000000000000001'
```
Comment (flopezag) [inline tests/units/test_change_password.py]: taking into account that OS_TENANT_ID is used several times in this file and in the test_openstackmap, I suggest to put like configuration variable of the unit tests execution.

**Comment by chemaper** [comment]
LGTM

File: tests/units/test_change_password.py (hunk 2 above)
Comment (chemaper) [inline tests/units/test_change_password.py]: But I suppose this is only a fake value. Isn't it? In this case I don't think a configuration variable make sense.

Comment (flopezag) [inline tests/units/test_change_password.py]: if it is no sense to put into a configuration variable put it as a common variable either in test_change_password.py or in the other one (i prefer the other due to it is where to make the comparison).

File: skuld/change_password.py (hunk 1 above)
Comment (geonexus) [inline skuld/change_password.py]: fixed

File: tests/units/test_change_password.py (hunk 2 above)
Comment (chemaper) [inline tests/units/test_change_password.py]: Ok

**Comment by flopezag** [comment]
LGTM

**Pull request merged**